subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -101,7 +101,7 @@ abstract class AbstractSmokeTest extends Specification {
         static androidGradle = Versions.of(*AGP_VERSIONS.latestsPlusNightly)
 
         // https://search.maven.org/search?q=g:org.jetbrains.kotlin%20AND%20a:kotlin-project&core=gav
-        static kotlin = Versions.of(*KOTLIN_VERSIONS.latests.findAll { !it.contains('-M')})
+        static kotlin = Versions.of(*KOTLIN_VERSIONS.latests.findAll { !it.contains('-M') && !it.contains('-RC') })
```

Comment:
Should we also check for `-EAP`?

Reply:
I looked what versions have been published up to now. I only found `-m`, `-rc` and `-beta` in different casing. I am now comparing lowercase strings and added the check fo `-beta` as well.